src/pages/ReportDetailsPage.tsx

```diff
@@ -645,10 +645,10 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
                 <ScrollView style={[styles.flex1]}>
                     <View style={[styles.reportDetailsTitleContainer, styles.pb0]}>
                         {renderedAvatar}
-                        {isExpenseReport && !shouldShowTitleField && nameSectionExpenseIOU}
+                        {isExpenseReport && (!shouldShowTitleField || !titleField) && nameSectionExpenseIOU}
```

Comment:
Hey, I'm just wondering why we only need to wrap these two books here and not below?

Reply:
Ah thank you